src/v2/System/useFeatureFlag.tsx

```diff
@@ -55,36 +51,40 @@ export function useTrackVariantView({
   experimentName,
   variantName,
   payload,
-  contextOwnerType,
-  contextOwnerId,
-  contextOwnerSlug,
-  shouldTrackExperiment = true,
 }: VariantTrackingProperties) {
-  const { trackEvent } = useTracking()
-
-  useEffect(() => {
-    if (typeof window !== "undefined") {
-      const trackFeatureView = shouldTrack(experimentName, variantName)
-
-      if (trackFeatureView && shouldTrackExperiment) {
-        trackEvent({
-          action: ActionType.experimentViewed,
-          service: "unleash",
-          experiment_name: experimentName,
-          variant_name: variantName,
-          payload,
-          context_owner_type: contextOwnerType,
-          context_owner_id: contextOwnerId,
-          context_owner_slug: contextOwnerSlug,
-        })
-      }
+  const router = useRouter()
+
+  const trackVariantView = () => {
+    // HACK: Temporary hack while we refactor useAnalyticsContext
+    // to update upon page navigation.
```

Comment:
Work to resolve this has started in this [branch](https://github.com/artsy/force/tree/mc-jones/refactor-analytics-context)